storage/src/cache/worker.rs

```diff
@@ -214,7 +214,7 @@ impl AsyncWorkerMgr {
                         .fetch_add(1, Ordering::Relaxed);
 
                     with_runtime(|rt| {
-                        rt.block_on(Self::handle_prefetch_requests(mgr2.clone(), rt));
+                        rt.block_on(Self::handle_prefetch_requests(mgr2.clone(), &ASYNC_RUNTIME));
```

Comment:
In fact, I agree this PR is kind of ugly or hacky since it is just a workaround or a demonstration of how we can get rid of tokio warnings. It is not a perfect or suitable solution now, which I was not planning to publish.  Let's suspend this PR for a while until a more elegant way to fix the tokio warning? I will raise an issue marking it  and close this PR